api/cas-server-core-api-configuration-model/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -0,0 +1,136 @@
+{
+  "properties": [
+    {
+      "name": "cas.authn.releaseProtocolAttributes",
+      "type": "java.lang.Boolean",
+      "description": "Whether to release authentication attributes.",
+      "defaultValue": true,
+      "deprecation": {
+        "replacement": "cas.authn.authenticationAttributeRelease.enabled",
+        "reason": "Property renamed due to new cas.authn.authenticationAttributeRelease.* settings.",
+        "level": "error"
+      }
+    },
+    {
+      "name": "cas.authn.x509.principalSNRadix",
+      "type": "java.lang.Integer",
+      "description": "X509 principal extraction config.",
+      "defaultValue": 10,
+      "deprecation": {
+        "replacement": "cas.authn.x509.serialNo.principalSNRadix",
+        "reason": "Property renamed due to x509 settings reorganization.",
+        "level": "error"
+      }
+    },
+    {
+      "name": "cas.authn.x509.principalHexSNZeroPadding",
+      "type": "java.lang.Boolean",
+      "description": "X509 principal extraction config.",
+      "defaultValue": true,
+      "deprecation": {
+        "replacement": "cas.authn.x509.serialNo.principalHexSNZeroPadding",
+        "reason": "Property renamed due to x509 settings reorganization.",
+        "level": "error"
+      }
+    },
+    {
+      "name": "cas.authn.x509.valueDelimiter",
+      "type": "java.lang.String",
+      "description": "X509 principal extraction config.",
+      "defaultValue": ",",
+      "deprecation": {
+        "replacement": "cas.authn.x509.serialNoDn.valueDelimiter",
+        "reason": "Property renamed due to x509 settings reorganization.",
+        "level": "error"
+      }
+    },
+    {
+      "name": "cas.authn.x509.serialNumberPrefix",
+      "type": "java.lang.String",
+      "description": "X509 principal extraction config.",
+      "defaultValue": "SERIALNUMBER=",
+      "deprecation": {
+        "replacement": "cas.authn.x509.serialNoDn.serialNumberPrefix",
+        "reason": "Property renamed due to x509 settings reorganization.",
+        "level": "error"
+      }
+    },
+    {
+      "name": "security.user.name",
```

Comment:
You should be able to remove these; These are boot specific-settings that are reported and/or migrated by the native Boot facility. We should not have to maintain them separately.

Reply:
I removed those and a webapp.gradle change I accidentally checked in, should I go ahead and merge or do you want to?

Reply:
Please do.